### DISCUSSION TITLE
Zim 0.75 released !

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#2162] [Announcements]
Dear all,

Zim release 0.75 is now available. It took quite a bit longer than I had expected (as usual, so much for estimations...). 

Highlights are the reworked interface for the tasklist plugin and updates to a number of other plugins (see the log below).

Less noticeable, but a big chunk of work was the refactoring of internal APIs for filesystem access and parsetree handling. These should result in better stability and easier maintenance going forward.

Please help updating the translations here: https://hosted.weblate.org/projects/zim/develop/ (notice we are now translating on the "develop" branch) and I'll release a 0.75.1 version in a month or so with all translation updates.

Regards,

Jaap

##  0.75.0 - Tue 27 Sep 2022
* Upgrade Python version requirement to >= 3.6
* Complete porting all modules to `zim.newfs` and deprecate `zim.fs`
* Refactor APIs to deal with tokens instead of `etree` interface
* Add framework for imagegenerators with inline data
* Refactored ther user interface of the tasklist plugin
* Make tasklist window persistent
* Let tasklist window also show closed tasks
* Let tasks inherit all modifiers from list heading
* Let "Go to Today" use anchor id for current day
* Update LinkMap with new options
* Refactor preference for custom window decoration & toolbar plugin
* Enables distinct shortcuts for en and em dashes in symbol list
* Add unicode 8757 ("because") in symbol list
* Move Selected Text: add preference for short links & save bounds
* Extend ToC pane context menu
* Score editor: add check for convert-ly
* Source View plugin: Don't show hidden languages
* Export code blocks in markdown as fenced blocks
* Fix loading GIF images
* Fix to let the "insert link" dialog allow anchors
* Allow sidepane focus key to be configured
* Added action to collapse and expand in PageIndex
* Add action for "move text" to allow keybinding
* Support closing an open 'find bar' by pressing ESC in the page view widget
* Allow cancelling ImageGeneratorDialog using Escape key
* Version control: fix fossil
* Fix KDE drag-n-drop bug
* Fix export with formatting in heading
* Minimal support for "Terminal" option in desktop files
* Add preference for not prompting on new folder

**Comment by dehesselle** [comment]
The macOS app has just been released.

**Comment by menthalo37** [comment]
Thanks Jaap, really good news for an excellent product (even if I've often issues to (re) install it with full plugins (uml through plantuml, equation ~editors) functionalities.
